components/automate-ui/src/app/modules/infra-proxy/policy-groups/policy-groups.component.html

```diff
@@ -18,8 +18,29 @@
     <chef-loading-spinner class="full-screen-spinner" *ngIf="searching" size="50" fixed></chef-loading-spinner>
     <div id="policy-group-table-container" *ngIf="policyFiles.length" data-cy="policy-group-table-container">
       <app-policy-groups-list
-        [policyFiles]= "groupList">
+        *ngIf="!searchFlag"
+        [policyFiles]= "policyGroups"
```

Comment:
name should be with group no policy files we are showing so it should be group on left hand side

Reply:
updated changes